pytd/writer.py

```diff
@@ -442,6 +451,11 @@ def _fetch_td_spark(self, apikey, endpoint, td_spark_path, download_if_missing):
             site = "eu01"
         conf.set("spark.td.site", site)
 
+        if spark_configs is None:
```

Comment:
```py
 if spark_configs is not None:
        for k, v in spark_configs.items():
            conf.set(k, v)
```

or, just set an empty dict by default: `def __init__(self, ..., spark_configs={}):`

Reply:
Initially, I implemented as you mentioned, but found that setting a mutable variable as a default argument introduces problematic behavior:
https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
I see. How about the conditional statement? Personally creating dummy dict was just confusing...

Reply:
Ok, I employ your suggestion bac9e60